x-pack/plugins/cases/server/client/utils.test.ts

```diff
@@ -916,4 +920,36 @@ describe('utils', () => {
       });
     });
   });
+
+  describe('constructSearchById', () => {
+    const savedObjectsSerializer = createSavedObjectsSerializerMock();
+
+    it('handles uuid-v4 search terms correctly', () => {
+      const uuidv4 = '0d397548-30f7-4a7f-aa54-cfe1a38cb902';
+
+      expect(constructSearch(uuidv4, DEFAULT_NAMESPACE_STRING, savedObjectsSerializer))
+        .toMatchInlineSnapshot(`
+        Object {
+          "rootSearchFields": Array [
+            "_id",
+          ],
+          "search": "\\"0d397548-30f7-4a7f-aa54-cfe1a38cb902\\" \\"cases:0d397548-30f7-4a7f-aa54-cfe1a38cb902\\"",
+        }
+      `);
+    });
+
+    it('handles non-uuid-v4 search terms correctly', () => {
```

Comment:
nit: same here something like `leaves the search value the same as passed was passed in`